.projen/files.json

```diff
@@ -7,6 +7,8 @@
     ".github/workflows/publish-release.yml",
     ".github/workflows/test.yml",
     ".gitignore",
+    ".husky/check-commit-msg.js",
```

Comment:
@quesabe why are there husky changes in the diff?

Reply:
I guess they came from an improper husky-related update. I'll move these into a separate PR, in order to make the history more clear.